templates/home.html

```diff
@@ -0,0 +1,28 @@
+<html lang="en">
+  <head>
+    <style>
+      .button {
+        background-color: #000000;
+        border: none;
+        color: white;
+        padding: 20px 34px;
+        text-align: center;
+        text-decoration: none;
+        display: inline-block;
+        font-size: 20px;
+        margin: 4px 2px;
+        cursor: pointer;
+      }
+      .button:hover {
+      background-color: #FFFFFF;
+      color: black;
+      }
+    </style>
```

Comment:
Seems like this is shared among all templates. Can we put it in a separate file and just load it as a css stylesheet?
A base template is also a good idea to separate all common page elements.

Reply:
I added a stylesheet to take care of these common style elements